app/models/container_node.rb

```diff
@@ -79,4 +79,18 @@ def ipaddress
   def cockpit_url
     URI::HTTP.build(:host => ipaddress, :port => 9090)
   end
+
+  def tenant_identity
+    if ext_management_system
+      ext_management_system.tenant_identity
+    else
+      User.super_admin.tap { |u| u.current_group = Tenant.root_tenant.default_miq_group }
+    end
+  end
```

Comment:
I wish we could stop copy-pasting this black magic, but that would take someone who understands it to suggest an appropriate place...